index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>API Project</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Nunito" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Indie+Flower" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+
+</head>
+<script id="usersData-temp" type="text/x-handlebars-template">
+    <div class="user-info">
+        <img id="profile-pic" src="{{pic}}">
+        <div class="user-data">
+            <p>{{first}} {{last}}</p>
+            <p>{{city}}, {{state}}</p>
+        </div>
+    </div>
+</script>
+<script id="friendsData-temp" type="text/x-handlebars-template">
+    {{#if this.length}}
+        <ul>
+        {{#each this}}
+            <li>{{first}} {{last}}</li>
+        {{/each}}
+        </ul>
+    {{/if}}
+</script>
+<script id="quotesData-temp" type="text/x-handlebars-template">
+    {{#if this.length}}
+        <p>Favorite quote:</p>
+        {{#each this}}
+            <span>"{{capitalize quote}}"<br>- Kanye</span>
+        {{/each}}
+    {{/if}}
+</script>
+<script id="baconsData-temp" type="text/x-handlebars-template">
+    {{#if this.length}}
+        <p class="meat-text">About me:</p>
+        {{#each this}}
+            <p class="meat-text">{{capitalize bacon}}</p>
+        {{/each}}
+    {{/if}}
+</script>
+<script id="pokemonsData-temp" type="text/x-handlebars-template">
+    {{#if this.length}}
+        {{#each this}}
+            <img id="pokemon-image" src="{{image}}">
+            <p id="pokemon-text">Favorite Pokemon: {{capitalize text}}</p>
+        {{/each}}
+    {{/if}}
+</script>
+<body>
+    <div class="container">	
```

Comment:
all your handlebar scripts should go at the bottom of you html right before the jquery script